examples/repl.html

```diff
@@ -7,8 +7,8 @@
     <title>REPL</title>
 
     <link rel="icon" type="image/png" href="favicon.png" />
-    <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
-    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
+    <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript_base.css" />
```

Comment:
probably missed here? + is `pyscript_base.css` is the correct file?

Reply:
Good eye, thanks. Not sure what the pyscript_base thing is...changed it.